utopia-vscode-extension/src/extension.ts

```diff
@@ -141,8 +141,25 @@ function didClose(path: string): DidClose {
 
 type SubscriptionWork = UpdateDirtyContentChange | DidChangeTextChange | WillSaveText | DidClose
 let pendingWork: Array<SubscriptionWork> = []
+let pendingWorkHandle: number
 
-function minimisePendingWork(): void {
+function pushWorkToQueue(work: SubscriptionWork) {
```

Comment:
This was to remove an unnecessary busy wait, which I brought up in the original PR